appinfo/info.xml

```diff
@@ -19,13 +19,9 @@ For more information, please read the [ownCloud Documentation](https://doc.owncl
 	<screenshot>https://raw.githubusercontent.com/owncloud/screenshots/master/templateeditor/ownCloud-app-email-template-editor.jpg</screenshot>
 	<namespace>TemplateEditor</namespace>
 	<dependencies>
-		<owncloud min-version="10.2" max-version="10" />
+		<owncloud min-version="4" max-version="100" />
```

Comment:
We could also remove the `<documentation>` section, and replace the `<description>` section with words that say that the template editor app has been discontinued (or whatever are the right words).
But that could be done in the release process - when the `<version>` section should be bumped to `0.5.0` etc.

Reply:
I will do that right away. Thx for noticing